data/hails.txt

```diff
@@ -2479,6 +2479,38 @@ phrase "hostile bounty"
 		"off beam."
 		"mislead."
 
+
+phrase "friendly quarg"
+	word
+		"We"
+		"The Quarg"
+		"Our people"
+	word
+		" "
+	word
+		"guide and guard"
+		"watch"
+		"oversee"
+		"look after"
+		"safeguard"
+	word
+		" "
+	word
+		"the galaxy"
+		"the young and foolish"
+		"the young races"
+		"the galaxy's children"
+		"those who live"
+	word
+		" "
+	word
+		"as we are sworn to do."
+		"as the Drak guide us."
+		"to uphold the sanctity of life."
+		"as is our geis."
```

Comment:
I'm of the opinion that geis fits the Quarg's rather antiquated speech patterns better.